script/Ciel/Day 14/14_01_CIEL14_1B.txt

```diff
@@ -322,7 +322,7 @@ My throat won't move.
 -- Page 59, Offset 37196.
 -- そんな余分な真似をすれば、次の瞬間、この首が千切られて転がる予感が、全身を縛って離れない。
 I remain frozen in place,
- paralyzed by the intuition that if I do anything so pointless as breathe then my head will be ripped clean off my shoulders.
+ paralyzed by the intuition that if I do anything so pointless as breathe, then my head will be ripped clean off my shoulders.
```

Comment:
Damn... this sentence really bothers me

Reply:
any suggestions on what to change it to?